.github/workflows/ci.yml

```diff
@@ -11,6 +11,10 @@ concurrency:
   group: ci-${{ github.head_ref || github.run_id }} # will be canceled on subsequent pushes in pull requests but not branches
   cancel-in-progress: true
```

Comment:
> Auto-review tests were moved to SCA workflow, which is also a little weird since these are... tests 😅.

Auto-review, I was bringing this idea years ago. Dream-setup for it was to have dedicated, extracted tool, with each review-type be separated "rule" (yeah, we see it like a Fixer-approach thingy).

yet, with lack of capacity, I used tool I had available cheaply - PHPUnit runner.
yet, we do not treat it as tests. We don't run it on each env, we don't collect coverage, we don't...

Reply:
So maybe we should verify if one of these tools can do it instead of unit tests:
- [Arkitect](https://github.com/phparkitect/arkitect)
- [PHPat](https://github.com/carlosas/phpat)
- [PHPStan](https://github.com/phpstan/phpstan) with custom rules?

But for me it's not a priority since these auto-review tests do their job well 🙂.

Reply:
first 2 could be interesting option, for PHPStan it's not - it's more SCA like Fixer than to check if half-generated docs are in sync, etc.

anyway, agree that potential tool migration is out of scope, and not sure if worth the effort (vs priority), but if you agree seeing the concern AutoReview doing to be close to Arkitect or PHPat, then we do not need to treat them as regular tests. https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7407/commits/b15502a6d52ef68282097f595633a09cd9118c97